android-components/components/lib/crash/src/main/java/mozilla/components/lib/crash/CrashReporter.kt

```diff
@@ -243,7 +246,25 @@ class CrashReporter(
 
         val additionalIntent = Intent()
         crash.fillIn(additionalIntent)
-        nonFatalCrashIntent?.send(context, 0, additionalIntent)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
+            val onFinished = null
+            val handler = null
+            val requiredPermission = null
+            val activityOptions = ActivityOptions.makeBasic()
+            activityOptions.pendingIntentBackgroundActivityStartMode =
```

Comment:
@rocketsroger @Amejia481 was this `activityOptions` object supposed to be used for something? I don't see it connected to the crash intent in any way.

Reply:
Maybe I missed understood, but I think we need for the notification pops up, when we crash, which could allow users to restart Fenix. @rocketsroger could you help to clarify? :)

Reply:
Sorry, my bad I was supposed to pass it to `send`, I'll put a PR for addressing it.

Reply:
We made the decision to add it, as we are under short deadline for supporting Android 14 we prefer to have it in case we missed a use case that requires it , and eventually this patches will be uplift to release.